contributions.md

```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
 4. Additional content describing common pitfalls, challenges, or limitations teams commonly encounter when applying certain practices or capabilities.
 5. Typos or grammatical fixes.
 
-If you chose to contribute a new practice try to follow our pre-established structure:
+If you chose to contribute a new practice, please try to follow our pre-established structure:
 
 ```markdown
-# Title Of Practice
+# Title of Practice
 
-A 2-3 sentence description of the practice followed by a sentence or two describing the main benefits of adopting this practice. Try to keep it brief.
+A 2-3 sentence description of the practice, followed by a sentence or two describing the main benefits of adopting this practice. Try to keep the text brief, relatable, and motivational.
```

Comment:
Would you like to also clarify expectations/style regarding the level of technical detail in these Practices? If so, we can add a line like this: "Our audience wants a quick read. The writing style should be conversational, not academic; the details should be higher level (think overview or summary), not too technical."

Reply:
yes please. The text you have works great.